sql/src/test/java/io/crate/planner/consumer/NestedLoopConsumerTest.java

```diff
@@ -197,6 +197,18 @@ public void testLeftSideIsBroadcastIfLeftTableIsSmaller() throws Exception {
         assertThat(collect.collectPhase().distributionInfo().distributionType(), is(DistributionType.BROADCAST));
     }
 
+    @Test
+    public void testRightSideIsNotAlwaysNullIfLeftTableIsSmaller() {
```

Comment:
How are we sure that **users** is smaller than **users_multi_pk**?

Reply:
Have a look at the `prepare()` method. We also don't know in other tests. The goal is to test that we don't `null` a `MergePhase` when we have table stats enabled. That's a bit like testing a test but I can include an assertion for the table stats.

Reply:
Thx! didn't see that before, sorry.